docs/src/less/dox-chrome.less

```diff
@@ -27,8 +27,8 @@
 		z-index: 0 !important;
 		.ts-toolbar-title label {
 			-webkit-font-smoothing: antialiased;
-			font-family: Tradeshifticons;
-			font-size: @ts-fontsize-xxbig + 1px;
+			// font-family: Tradeshifticons;
```

Comment:
delete it, when you don't want to use it anymore.

Reply:
Removed.